filebeat/input/filestream/config.go

```diff
@@ -69,9 +69,11 @@ type readerConfig struct {
 	MaxBytes       int                     `config:"message_max_bytes" validate:"min=0,nonzero"`
 	Tail           bool                    `config:"seek_to_tail"`
 
-	Parsers []*common.ConfigNamespace `config:"parsers"` // TODO multiline, json, syslog?
+	Parsers ParserConfig `config:"parsers"`
 }
 
+type ParserConfig []*common.ConfigNamespace
```

Comment:
There is no need for a pointer here I think. `[]common.ConfigNamespace` should do the trick.